admin_guide/available_ports.adoc

```diff
@@ -0,0 +1,117 @@
+= Available Ports
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+OpenShift infrastructure components talk to each other using ports;
+communication endpoints that are identifiable for specific processes or
```

Comment:
Maybe I'm just not familiar w/ this type of semi-colon usage, but shouldn't the part after the semi-colon be a complete sentence?